.github/scripts/crib/pr-comment-crib-env.js

```diff
@@ -57,6 +88,31 @@ async function run() {
       return;
     }
 
+    // Check if DNS records exist in Route 53 before printing out the subdomains.
+    try {
+      const maxRetries = 8;
+      const recordsExist = await route53RecordsExist(
+        route53ZoneId,
+        subdomains,
+        maxRetries
+      );
+      if (recordsExist) {
+        core.info("Route 53 DNS records exist:", recordsExist);
+      } else {
+        core.setFailed(
+          "Route 53 DNS records do not exist. Please check the Route 53 hosted zone."
+        );
+        return;
+      }
```

Comment:
```suggestion
      } 
        core.setFailed(
          "Route 53 DNS records do not exist. Please check the Route 53 hosted zone."
        );
        return;
     
```

Reply:
If I move this out of the conditional then if `recordsExist` is true, it will print

`core.info("Route 53 DNS records exist:", recordsExist);`

And then run

```
core.setFailed(
          "Route 53 DNS records do not exist. Please check the Route 53 hosted zone."
        );
        return;
```

I suppose we could invert this so we check if `recordsExist` is false and then do the core.setFailed() with the return. After that block we could do the core.info() but at this point, I might just leave it?

Reply:
oh my bad, yeah i was gunning for an inversion. I find it reduces complexity when you remove nesting. This is more of a nit. Feel free to leave or change

Reply:
All good, thanks for all of these suggestions. I'll probably take the easy way out here and leave it then 👍